chart-infra/templates/pod-template.tpl

```diff
@@ -50,7 +50,7 @@
         resources:
           requests:
             memory: "1Gi"
-{{- if eq .Values.myAccount.datadogEnabled "true" }}
+{{- if eq .Values.clusterEnv "production" }}
```

Comment:
I believe this will break HMS as we don't use datadog. I previously had to restore this  to stop worker pods from crashing